<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>ADC Readback</string>
    </property>
    <widget class="QTableView" name="tv_adc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>431</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>691</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>DAC Write</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Channe 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_dac_ch1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_dac_ch2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_dac_ch3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_dac_ch4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_dac_update_0">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_dac_update_1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_dac_update_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_dac_update_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_dac_use_raw">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Raw Data</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>451</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Resolver</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Channe 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_resolver_pos1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_resolver_pos2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_resolver_vecl1">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_resolver_vecl2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>721</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Serial Port</string>
    </property>
    <widget class="QComboBox" name="cbtty">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ttyMAX0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyMAX1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyMAX2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyMAX3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttymxc3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttymxc4</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="le_serial_data">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>451</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="te_serial_data">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>451</width>
       <height>70</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_com_send">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>40</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_port_open">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>420</y>
      <width>721</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Digital I/O</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TTL:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_ttl">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_iso_di">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ISO In</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_iso_out_0">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>ISO OUT 1</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_iso_out_1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>ISO OUT 2</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_iso_out_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>ISO OUT 3</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_iso_out_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>180</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>ISO OUT 4</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionADC_Test"/>
   <addaction name="actionDAC_Test"/>
   <addaction name="actionResolver_Test"/>
   <addaction name="actionDIO_Test"/>
  </widget>
  <action name="actionADC_Test">
   <property name="text">
    <string>ADC Test</string>
   </property>
  </action>
  <action name="actionDAC_Test">
   <property name="text">
    <string>DAC Test</string>
   </property>
  </action>
  <action name="actionResolver_Test">
   <property name="text">
    <string>Resolver Test</string>
   </property>
  </action>
  <action name="actionDIO_Test">
   <property name="text">
    <string>DIO Test</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
